### ISSUE TITLE
Fix broken external links in documentation

### ISSUE THREAD

**Issue by themr0c** [eclipse-che/che#19841]
####    Cause

What actions or circumstances cause this bug to present.

These URLs are not accessible anymore:

*   https://github.com/eclipse/che/tree/master/dockerfiles/postgres
*   https://github.com/eclipse/che/tree/master/dockerfiles/keycloak
* https://github.com/eclipse/che/blob/master/deploy/openshift/templates/monitoring/prometheus-config.yaml
* https://github.com/eclipse/che/blob/master/devfile.yaml

####    Consequence

What happens when the bug presents.

Documentation builds fail to validate links

```
Run htmltest
htmltest started at 06:36:44 on build/site/che-7
========================================================================
administration-guide/che-workspace-controller/index.html
  Non-OK status: 404 --- administration-guide/che-workspace-controller/index.html --> https://github.com/eclipse/che/tree/master/dockerfiles/postgres
  Non-OK status: 404 --- administration-guide/che-workspace-controller/index.html --> https://github.com/eclipse/che/tree/master/dockerfiles/keycloak
administration-guide/monitoring-che/index.html
  Non-OK status: 404 --- administration-guide/monitoring-che/index.html --> https://github.com/eclipse/che/blob/master/deploy/openshift/templates/monitoring/prometheus-config.yaml
end-user-guide/authoring-devfiles-version-1/index.html
  Non-OK status: 404 --- end-user-guide/authoring-devfiles-version-1/index.html --> https://github.com/eclipse/che/blob/master/devfile.yaml
end-user-guide/creating-a-workspace-from-local-devfile-using-chectl/index.html
  Non-OK status: 404 --- end-user-guide/creating-a-workspace-from-local-devfile-using-chectl/index.html --> https://github.com/eclipse/che/blob/master/devfile.yaml
========================================================================
✘✘✘ failed in 59.869891556s
5 errors in 371 documents
```

* See: https://github.com/eclipse/che-docs/pull/1985/checks?check_run_id=2632980115
* See: https://github.com/eclipse/che-docs/pull/1992/checks?check_run_id=2632979306

####    Fix

What was done to fix the bug.

Find the new URL for each broken link and replace it in the documentation.

####    Result

What now happens when the actions or circumstances above occur. Note that this is not the same as the bug doesn’t present anymore.

Links validation is successful.

### Che version

 - [ ] latest
 - [ ] nightly
 - [ ] other: please specify

### Runtime
   - [ ] kubernetes (include output of `kubectl version`)
   - [ ] Openshift (include output of `oc version`)
   - [ ] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Installation method
   - [ ] chectl
     * provide a full command that was used to deploy Eclipse Che (including the output)
     * provide an output of `chectl version` command
   - [ ] OperatorHub
   - [ ] I don't know

### Environment
   - [ ] my computer
       - [ ] Windows
       - [ ] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify)
   - [ ] Dev Sandbox (workspaces.openshift.com)
   - [ ] other: please specify

**Comment by themr0c** [comment]
Severity blocker as it blocks any PR merge.

**Comment by themr0c** [comment]
First problem that I see: we target master in the che repository, that is wrong.

**Comment by themr0c** [comment]
* Che server code moved to  https://github.com/eclipse-che/che-server
* That repository has tages for minor versions (but no 7.30.x branch for instance)
* We should move all links to https://github.com/eclipse/che

**Comment by mshaposhnik** [comment]
@themr0c PR fixing links ^^    Checks are passed

**Comment by themr0c** [comment]
Fixed by  eclipse/che-docs#2002
